Convert tiff package to DOWNLOAD helper usage.

No more needed with recent BR.

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

Index: buildroot/package/tiff/tiff.mk
===================================================================
--- buildroot.orig/package/tiff/tiff.mk	2010-11-22 13:46:28.000000000 +0100
+++ buildroot/package/tiff/tiff.mk	2010-11-22 13:48:03.000000000 +0100
@@ -11,7 +11,7 @@
 TIFF_CAT:=$(ZCAT)
 
 $(DL_DIR)/$(TIFF_SOURCE):
-	 $(WGET) -P $(DL_DIR) $(TIFF_SITE)/$(TIFF_SOURCE)
+	$(call DOWNLOAD,$(TIFF_SITE),$(TIFF_SOURCE))
 
 tiff-source: $(DL_DIR)/$(TIFF_SOURCE)
 
